extensions/vscode/src/events.ts

```diff
@@ -177,15 +177,24 @@ export class EventStream extends Readable implements Disposable {
   }
 
   private processMessage(msg: EventStreamMessage) {
-    const localId = msg.data.localId;
-    if (localId && this.canceledLocalIDs.includes(localId)) {
+    // Some log messages passed on from Connect include
+    // the localId using snake_case, rather than pascalCase.
+    // To filter correctly, we need to check for both.
+
+    const localIdForm1 = msg.data.localId;
```

Comment:
If it is always one or the other can we grab either one?

```suggestion
    const localIdForm1 = msg.data.localId || msg.data.local_id;
```

Reply:
first one which is defined, I think. We'll confirm as we debug the issue on your machine. I've changed the code to use your suggestion (it was a good one).